
Don't let pgtcl lower down our tunning flags from shared-macros.mk
not suitable for upstream.

--- Pgtcl-3.1.0/aclocal.m4.orig	2024-05-13 17:47:36.000000000 +0200
+++ Pgtcl-3.1.0/aclocal.m4	2025-03-15 09:22:12.156679763 +0100
@@ -945,8 +945,8 @@
 			    AC_MSG_WARN([64bit mode not supported with GCC < 3.2 on $system])
 			], [
 			    do64bit_ok=yes
-			    CFLAGS="$CFLAGS -m64 -mcpu=v9"
-			    LDFLAGS="$LDFLAGS -m64 -mcpu=v9"
+			    CFLAGS="$CFLAGS -m64 -mcpu=ultrasparc"
+			    LDFLAGS="$LDFLAGS -m64 -mcpu=ultrasparc"
 			    SHLIB_CFLAGS="-fPIC"
 			])
 		    ], [
@@ -996,7 +996,7 @@
 			# add the path to the sparv9 libgcc.
 			# JH: static-libgcc is necessary for core Tcl, but may
 			# not be necessary for extensions.
-			SHLIB_LD="$SHLIB_LD -m64 -mcpu=v9 -static-libgcc"
+			SHLIB_LD="$SHLIB_LD -m64 -mcpu=ultrasparc -static-libgcc"
 			# for finding sparcv9 libgcc, get the regular libgcc
 			# path, remove so name and append 'sparcv9'
 			#v9gcclibdir="`gcc -print-file-name=libgcc_s.so` | ..."
